internal/image/image_installer.go

```diff
@@ -109,7 +109,7 @@ func (img *ImageInstaller) InstantiateManifest(m *manifest.Manifest,
 	rootfsImagePipeline := manifest.NewISORootfsImg(m, buildPipeline, anacondaPipeline)
 	rootfsImagePipeline.Size = 4 * common.GibiByte
 
-	bootTreePipeline := manifest.NewEFIBootTree(m, buildPipeline, anacondaPipeline)
+	bootTreePipeline := manifest.NewEFIBootTree(m, buildPipeline, img.Product, img.OSVersion)
```

Comment:
For the future: What's the semantics of `Product` and `OSVersion`? Do they describe the payload? Should the payload then be the one encoding them and expose them via some interface? If it is just a coincidence that they values are the same between the EFIBootTree and the Anaconda pipeline we should keep it as is done in this patch. Either way, no need to figure this out as part of this PR, this makes sense for now.

Reply:
I've been thinking about the same questions too and I don't have solid answers.  I made a lot of decisions here for technical simplicity, but I'd rather we figured out something that's more consistent semantically.

Reply:
Let's discuss it all when the dust settles ;)